### ISSUE TITLE
MRA: Some MRA is on MiSTer releases and in _alternatives at the same time

### ISSUE THREAD

**Issue by theypsilon** [jotego/jtbin#184]
These are:
```
Bubble Bobble (Japan, Ver 0.1).mra 
Contra (US 3).mra 
```

From where should we remove them? From `_alternatives`, or from `releases`?

Also, there is a mismatch between the MRA in `mra/bubl` which is `Bubble Bobble (Japan, Ver 0.1).mra` and `bubl/releases` which is `Bubble Bobble (US, Ver 5.1).mra`

These two folders should contain the same MRA file. Should we have `Bubble Bobble (Japan, Ver 0.1).mra` just in `mra/bobl` and `bobl/releases` in one side, and then move `Bubble Bobble (US, Ver 5.1).mra` in _alternatives? And for contra, should we just remove `Contra (US 3).mra` from `_alternatives` and add it to `mra/contra` for consistency?

**Comment by jotego** [comment]
Thank you. Fixed in https://github.com/jotego/jtbin/commit/72180ccf5dfacebe205a089a073c26c350c8f673
I just removed the ones in _alt.

**Comment by theypsilon** [comment]
Thanks @jotego !

There is still this mismatch:

On `mra/bubl` we got `Bubble Bobble (Japan, Ver 0.1).mra`
On `mister/bubl/releases` we got `Bubble Bobble (US, Ver 5.1).mra`

Since `mra/bubl` and `mister/bubl/releases` should contain the exact same MRAs, something still has to be changed there. Right now, `Bubble Bobble (US, Ver 5.1).mra` is the only one being distributed by the updater. I can suggest a solution later in PR form.
